serpent/utilities.py

```diff
@@ -54,6 +54,10 @@ def display_serpent_logo():
     """)
 
 
+def is_true(s):
```

Comment:
This is fine, but I would rather have in serpent.py, because it's the only logical place where we would do this check.